editor/store/effects.js

```diff
@@ -61,6 +62,38 @@ const TRASH_POST_NOTICE_ID = 'TRASH_POST_NOTICE_ID';
 const SAVE_REUSABLE_BLOCK_NOTICE_ID = 'SAVE_REUSABLE_BLOCK_NOTICE_ID';
 
 export default {
+	REQUEST_POST_AUTOSAVE( action, store ) {
```

Comment:
I believe this is really close to the `REQUEST_POST_UPDATE` action, I wonder if there's a way to factorize both, ( maybe by adding a flag to the `REQUEST_POST_UPDATE` action?

Reply:
probably - i separated it out for now because i wasn't too familiar with the other actions triggered and wanted to make sure i understood what was happening (originally i tried combining them).

I also realized we need two more things: * delete the autosave when the post is updated/saved (a revision is created) and * show the warning about an autosave existing on load when one does.

Reply:
I refactored this into REQUEST_POST_UPDATE